torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -12,7 +12,7 @@ struct UpgraderEntry {
   std::string old_schema;
 };
 
-const static std::unordered_map<std::string, std::vector<UpgraderEntry>> kOperatorVersionMap(
+static std::unordered_map<std::string, std::vector<UpgraderEntry>> kOperatorVersionMap(
```

Comment:
I think `k` prefix is only for constants.

Reply:
sounds good. will do

Reply:
done